docs/housing_search.md

```diff
@@ -12,7 +12,7 @@ ElasticSearch will be updated by a Lambda function, holding business logic, that
 
 ## Vision
 
-- A reliable and efficient search service, that allows for free-text and fuzzy search for people and properties.
+- A reliable and efficient search service, that allows for free-text and fuzzy/exact search for people and properties.
```

Comment:
Please explain explicitly the need for the exact match for address 1 as the data in address 1 can sometimes be unreliable..

Reply:
Added